ecocode-rules-specifications/src/test/java/fr/greencodeinitiative/RulesTest.java

```diff
@@ -0,0 +1,263 @@
+/*
```

Comment:
I'm ok to create a new Test class to check all files and all rules. but I think we can do it simplier, maybe.
I will run your test class and give my feedback soon.

Reply:
@dedece35 Did you have time to review the test class?